README.md

```diff
@@ -104,16 +104,14 @@ sudo yum install make cmake gcc-c++ scons
 sudo yum install libmicrohttpd-devel boost-devel libcurl-devel
 ```
 
-* Install the Mongo Driver from source (reference procedure http://docs.mongodb.org/ecosystem/tutorial/getting-started-with-cpp-driver/):
+* Install the Mongo Driver from source:
 
 ```
-wget http://downloads.mongodb.org/cxx-driver/mongodb-linux-x86_64-2.2.3.tgz
-# Don't pay attention to the '-linux-x86_64' in the name: it is actually source code for any platform
-tar xfvz mongodb-linux-x86_64-2.2.3.tgz
-cd mongo-cxx-driver-v2.2
-scons                                         # The build/libmongoclient.a library is generated as outcome
-sudo scons install                            # This puts .h files in /usr/local/include and libmongoclient.a in /usr/local/lib
-sudo chmod a+r -R /usr/local/include/mongo    # It seems that scons install breaks permissions
+wget https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.0.2.tar.gz
```

Comment:
This is a slight variant of the procedure @crbrox sent by email (based on "git checkout legacy"), in order to ensure that we stick to a particular version rather than to the "moving picture" of a git branch.

NTC (just informative)